tests/server/test_notifications.py

```diff
@@ -415,7 +415,7 @@ async def notification_cleaned_up(env_id: uuid.UUID, expected_length_after_clean
 
 @pytest.mark.parametrize("halted", [True, False])
 @pytest.mark.parametrize("no_agent", [True])
-async def test_cleanup_notifications(server, client, halted_env_with_old_notifications, halted):
+async def test_cleanup_notifications(server, client, halted_env_with_old_notifications, halted, no_agent: bool):
```

Comment:
This doesn't seem to be used in the test ? Or am I missing something ? (same in a couple of other places)

Reply:
My IDE was complaining that this argument was not given, whereas the decorator provides it

Reply:
Right I see, let's leave it up for the big test review